content/studies/sample.md

```diff
@@ -24,12 +24,101 @@ This is a pagraph — cillum nisi aliqua officia proident dolor esse sint ut non
 3. Ordered list item
 4. Ordered list item
 
+### One Image — simple markdown
+
 ![Field of sunflowers](/img/studies/light-two/sunflower.jpg)
 *Image with caption — `<em>` under the image*
 
-Quis deserunt ut irure irure dolore exercitation ipsum commodo Lorem Lorem enim. Qui qui sint deserunt nisi tempor Lorem proident. Laboris eiusmod aute nostrud nostrud duis eu sunt cupidatat ut pariatur consequat. Minim aliquip irure mollit voluptate adipisicing qui mollit labore sint anim velit sit sint. Fugiat est in fugiat occaecat voluptate ipsum Lorem elit ea ad ea elit pariatur labore. Elit cillum adipisicing voluptate esse sit. Laborum quis amet veniam minim cillum veniam in. Esse ut tempor nisi et eiusmod tempor ex. Laboris ullamco officia irure aute eu consectetur. Irure elit pariatur culpa eu culpa minim do non laboris mollit et incididunt aliqua exercitation. Fugiat adipisicing proident elit consectetur dolore est ut et fugiat non aliquip occaecat mollit ad. Non dolore do et sunt nulla cillum incididunt commodo incididunt esse.
+### Side-by-side modules
+div.cols > figure > img + figcaption
 
----
+<div class="cols">
+    <figure>
+        <img src="/img/studies/light-two/sunflower.jpg" alt="">
+        <figcaption>div.cols</figcaption>
+    </figure>
+    <figure>
+        <img src="/img/studies/light-two/phone-square.jpg" alt="">
+        <figcaption>div.cols</figcaption>
+    </figure>
+</div>
```

Comment:
syntax for adding side-by-side items. 
- note that we're using `<figcaption>` for captions here rather than `<em>`, which we are using for full-width images.